client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -675,6 +675,67 @@
 	}
 }
 
+.jetpack-checkout-siteless-thank-you-completed {
+	&.main.is-wide-layout {
+		max-width: 660px;
+	}
+	.jetpack-checkout-siteless-thank-you-completed__card {
+		padding: 0;
+		box-shadow: 0 0 40px 0 #00000014;
+
+		width: 100%;
+		max-width: none;
+
+		> div.jetpack-checkout-siteless-thank-you-completed__card-main {
+			box-sizing: border-box;
+			padding: 26px;
+			@include break-mobile {
+				padding: 76px;
+			}
+		}
+
+		.jetpack-logo {
+			margin-bottom: 48px;
+		}
+
+		h1 {
+			@include break-mobile {
+				font-size: $font-headline-medium;
+			}
+		}
+		p {
+			@include break-mobile {
+				font-size: 18px; /* stylelint-disable-line */
+				line-height: 1.75rem;
+			}
+		}
+
+		a.jetpack-checkout-siteless-thank-you-completed__link,
+		a.jetpack-checkout-siteless-thank-you-completed__feature-link {
+			text-decoration: underline;
+			color: var( --color-studio-black );
+			&:hover {
+				text-decoration: none;
+			}
+		}
+
+		.jetpack-checkout-siteless-thank-you-completed__main-message {
+			font-size: 2.25rem;
+			font-weight: 700;
+			margin-bottom: 56px;
+			line-height: 1;
+			@include break-mobile {
+				font-size: 3rem;
+				line-height: 1.25;
```

Comment:
To make this consistent with the other interfaces we've built, I'd remove this line.

However, I know a line-height of 1 is probably a bit _too_ condensed, but once we're all done with this we can go back and adjust all of them slightly to 1.1 (?).